Src/BootCamp.Chapter/PcFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BootCamp.Chapter.Computer;
+
+namespace BootCamp.Chapter
+{
+    public abstract class PcFactory
+    {
+        protected Body body;
```

Comment:
IMagine that your factory is really popular and you have multiple assembles happening at once. Do you really want to afford member fields storing info on what is being assembled? Preparing each part means creating it and attaching it to a PC. The flow is defined and known for both factories. Just individual steps are different.